Tests/FunctionalTests/TraitTests.swift

```diff
@@ -18,10 +18,10 @@ import XCTest
 final class TraitTests: XCTestCase {
     func testTraits_whenNoFlagPassed() async throws {
         try await fixture(name: "Traits") { fixturePath in
-            let (stdout, _) = try await executeSwiftRun(fixturePath.appending("Example"), "Example", extraArgs: ["--experimental-prune-unused-dependencies"])
+            let (stdout, stderr) = try await executeSwiftRun(fixturePath.appending("Example"), "Example", extraArgs: ["--experimental-prune-unused-dependencies"])
```

Comment:
Creating PR suggestions for every long line in this file is tedious, but I'd expect a SwiftFormat pass on this file to handle all long lines.